Port bitmap page updates from other repo

gbMichelle opened the pull request:
These changes were lost

csauve commented on src/content/blam/tags/bitmap/readme.md:
```diff
@@ -125,35 +113,40 @@ Used mostly for special effects this setting stores the rgb channels as directio
-[Images still need to be uploaded]
+|Scaled 10x | Original Size |
+|-----------|---------------|
+![](media/bitmap_color_plate.png)|![](media/bitmap_color_plate_orig_size.png)|
```
Images by default are styled as blocks with 100% width. That means this orig_size one is being shown scaled up and blurry. We need a solution here for image sizing and alignment within the content. E.g. should all images take up 100% width of their containers? Only when used in figures and metaboxes? What kind of standard image sizes do we want to support?

Doesn't have to be solved in this PR, but thought I'd point out that this image could use a style override in HTML as a workaround for now.

gbMichelle replied:
Yeah, this doesn't happen in github markdown. Which is what this version of the document was made for

gbMichelle replied:
A style override would be a good thing here.

csauve commented on src/content/blam/tags/bitmap/readme.md:
```diff
@@ -14,55 +14,43 @@ Bitmaps are used for visuals that need textures or sprites like environments, ob
-you dont need to have a multi for detail maps to show
-
-### Commands
+### Tool Commands
```
You have no h2 in this document -- h1 and h2 will show up in the ToC so I suggest changing these h3 to h2 where appropriate for easier navigation.

gbMichelle replied:
Looks like an oversight that there's only h1 and h3. I'd be fine changing it in either way

csauve replied:
The formats section would benefit from it especially

csauve commented on src/content/blam/tags/bitmap/readme.md:
```diff
@@ -192,15 +187,17 @@ _NOTE: Tool will sometimes split a sequence across several pages. This does not
-* **double multiply**: Makes the background grey (50% grey).
+ - **blend/add/subtract/max**: Makes the background black.
+ - **multiply/min**: Makes the background white.
+ - **double multiply**: Makes the background grey (50% grey).
```
Minor, but would you be OK removing these single space indents? I think it would play better with editors set up for a more usual 2 space indent.

gbMichelle replied:
Part of this was mainly readability in source form. But I am not super attached to the indents

csauve commented:
Regarding hard line wrapping at 80 characters, have you tried using these cursor navigation settings to make working on wrapped lines easier?

https://github.com/csauve/env-common/blob/master/.vimrc#L37

I’d be a bit concerned with the additional maintenance effort of 80 character hard wraps for something like free-form text. It would slow down the editing experience outside of editor plugin use, and if I wanted to make my editor narrower than 80 chars the formatting would make it hard to read. Single lines play better with diffs as well since text insertions won’t cause all subsequent lines of the paragraph to be modified as well.

gbMichelle commented:
Addressed bullet points and converted h3 to h2. Not sure what to do with the line size restriction comment at this time. Also not sure what to do about the style overrides.

csauve commented:
I'm inclined to merge this. Style improvements are something we can iterate on and maybe depend on us coming up with a writer's guide to know what kind of standards we want, and that doesn't need to block this content from going out. Ditto Mia's comment, given it wasn't described as a blocker.

I'll push these changes out to S3 locally.
